include/CXXGraphConfig.h

```diff
@@ -0,0 +1,4 @@
+// the configured options and settings for CXXGraph
+#define CXXGraph_VERSION_MAJOR 1
+#define CXXGraph_VERSION_MINOR 0
+#define CXXGraph_VERSION_PATCH 0
```

Comment:
We're checking in this file now.

Any change to the project version will be reflected here.